### ISSUE TITLE
Mobile decides

### ISSUE THREAD

**Issue by alandefreitas** [ultralight-ux/ultralight#42]
Do you guys have any plans to support mobile devices (iOS and Android)?

**Comment by adamjs** [comment]
Yes to Android, shouldn't be too much of a hassle. Will probably target OpenGL ES for the driver.

As for iOS, it might be a little tricky since we'll need to use their own copy of JavaScriptCore which may not be in sync with our own. I plan to research this more later.

**Comment by alandefreitas** [comment]
Are these plans in the near future?

**Comment by RPG3D** [comment]
Are these plans in the near future?+1

**Comment by mr5z** [comment]
Are these plans in the near future?+2

**Comment by adamjs** [comment]
Will begin work on the ARM port in about a month.

**Comment by cooldesigns** [comment]
Let's begin work together?

**Comment by WoLfulus** [comment]
How's it going? We'd use it if there's support for android/ios

**Comment by danielpaull** [comment]
It'd be nice to be able to send touch events to allow a rendered UI to work like a touch screen.

**Comment by omrta-dev** [comment]
So is there not any android support at the moment? Does this not work using the NDK ? I'm trying to see if this is a viable product for me but I'm not sure that it is without android support. I could work on the feature if needed but have to delve more into how much source code a free license provides.

**Comment by Osiris-Team** [comment]
I would like to contribute to add support for android and ios.
Is there anything I can do?
